### PR TITLE
Change senders name for emails

### LINKED ISSUE #3361: Change senders name for emails
### Please describe the problem (or idea)
image
I got the mail from notifications sounds ambiguous to me
> What happened just before the problem occurred? Or what problem could this idea solve?
Instead of notifications, ``Public Lab`` should be present.

> What did you expect to see that you didn't?
Instead of notifications, ``Public Lab`` should be present.

### What's your PublicLab.org username?
geekychaser

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

mozilla firefox
****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by igniteeng000**
Fixes #3361 
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by igniteeng000** [comment]
@publiclab/reviewers Please review

File: app/mailers/password_reset_mailer.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class PasswordResetMailer < ActionMailer::Base
   helper :application
   include ApplicationHelper
-  default from: "notifications@#{ActionMailer::Base.default_url_options[:host]}"
+  default from: "Public Lab <notifications@#{ActionMailer::Base.default_url_options[:host]}>"
 
   # PasswordResetMailer.reset_notify(user).deliver_now
   def reset_notify(user, key)
-    subject = '[Public Lab] Reset your password'
```
Comment (namangupta01) [inline app/mailers/password_reset_mailer.rb]: why are you removing this?

Comment (igniteeng000) [reply app/mailers/password_reset_mailer.rb]: Because there are two tasks given in this issue. Given by @SidharthBansal

Comment (igniteeng000) [reply app/mailers/password_reset_mailer.rb]: According to me it will not be necessary if we show public lab instead of notification

Comment (namangupta01) [reply app/mailers/password_reset_mailer.rb]: I am saying this why we need to remove this ?
@SidharthBansal why we need to remove this?

**Comment by namangupta01** [comment]
@igniteeng000 Please address the issue in the pr description like closes #....

**Comment by namangupta01** [comment]
@jywarren should we change the mail from `notifications` to something else because last time i remembered you said we should use `notifications` just like github do.

File: app/mailers/password_reset_mailer.rb (hunk 1 above)
Comment (igniteeng000) [reply app/mailers/password_reset_mailer.rb]: If it is showing the sender name is Public lab then no requirement of [Public Lab] agian in the title

Comment (igniteeng000) [reply app/mailers/password_reset_mailer.rb]: done

**Comment by SidharthBansal** [comment]
I think it is ambiguous to have notifications.
When we see gmail, we get msg from public lab not from notification

File: app/mailers/welcome_mailer.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class WelcomeMailer < ActionMailer::Base
   helper :application
   include ApplicationHelper
-  default from: "notifications@#{ActionMailer::Base.default_url_options[:host]}"
+  default from: "Public Lab <notifications@#{ActionMailer::Base.default_url_options[:host]}>"
```
Comment (jywarren) [inline app/mailers/welcome_mailer.rb]: I like this!

File: app/mailers/password_reset_mailer.rb (hunk 1 above)
Comment (jywarren) [reply app/mailers/password_reset_mailer.rb]: Here I think this is OK! See how things appear in Gmail - i think Public Lab will appear to the left:

image

**Comment by jywarren** [comment]
Great work @igniteeng000 and great issue writing @geekychaser!

**Pull request merged**